.gitignore

```diff
@@ -3,7 +3,7 @@ node_modules
 coverage
 coverage.json
 typechain
-yarn.lock
+package-lock.json
```

Comment:
`package-lock.json` should be removed from gitignore so if it is generated by mistake for anyone locally, it is easy to spot.

Reply:
makes sense, will do